testprojects/django111/testproject/settings.py

```diff
@@ -37,7 +37,6 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'django_embed_template',
```

Comment:
You should leave this one in (I'm open to other suggestions, but so far this seemed like the reasonable way to install the app)

Reply:
Right, I forgot to change that setting.